cpp/memilio/epidemiology/lct_infection_state.h

```diff
@@ -49,7 +49,7 @@ class LctInfectionState
      * @return Number of subcompartments for State.
      */
     template <InfectionState State>
-    static constexpr unsigned int get_num_subcompartments()
+    static constexpr int get_num_subcompartments()
```

Comment:
Since the return values can now be negative, please add to the documentation whether negative values can be returned.